samples/client/petstore/kotlin-string/src/main/kotlin/org/openapitools/client/apis/PetApi.kt

```diff
@@ -56,7 +56,7 @@ class PetApi(basePath: kotlin.String = "http://petstore.swagger.io/v2") : ApiCli
     * @param apiKey  (optional, default to null)
     * @return void
     */
-    fun deletePet(petId: kotlin.Long, apiKey: kotlin.String) : Unit {
+    fun deletePet(petId: kotlin.Long, apiKey: kotlin.String? = null) : Unit {
```

Comment:
@keijumt overall the change looks good.

Ideally should ` = null` be omitted (and only show the non-null default value)?

Reply:
@wing328 I Modified to omit `= null`.